hadoop-ozone/dist/src/main/license/update-jar-report.sh

```diff
@@ -30,4 +30,4 @@ cd "$OZONE_DIST_DIR"
 
 #sed expression removes the version. Usually license is not changed with version bumps
 #jacoco and test dependencies are excluded
-find . -type f -name "*.jar" | cut -c3- | perl -wpl -e 's/-[0-9]+(.[0-9]+)*(-SNAPSHOT)?+//g' | grep -v jacoco | grep -v hadoop-hdds-test-utils | sort > "$SCRIPTDIR"/$REPORT_NAME
+find . -type f -name "*.jar" | cut -c3- | perl -wpl -e 's/-[0-9]+(.[0-9]+)*(-SNAPSHOT)?+//g' | perl -wpl -e 's/[0-9a-z]+-SNAPSHOT//g' | grep -v jacoco | grep -v hadoop-hdds-test-utils | sort > "$SCRIPTDIR"/$REPORT_NAME
```

Comment:
Nit: the hash in snapshot jars could be handled by replacing `(-SNAPSHOT)?` with `(-([0-9a-f]+-)?SNAPSHOT)?`.  One less `perl` command, no ugly `-` suffix in jar names.

Reply:
Good idea, thanks for the suggestion. Pushed the fix in b9b7dae2c, will merge it once we have a green build.